packages/map-template/src/hooks/useMapBoundsDeterminer.js

```diff
@@ -62,61 +70,157 @@ const useMapBoundsDeterminer = () => {
     /*
      * When relevant state changes, run code to go to a location in the world.
      */
-    useEffect(() =>  {
+    useEffect(() => {
         determineMapBounds();
-    }, [mapsIndoorsInstance, currentVenueName, locationId, kioskOriginLocationId, pitch, bearing, startZoomLevel, categories]);
+    }, [mapsIndoorsInstance, currentVenueName, locationId, kioskOriginLocationId, pitch, bearing, startZoomLevel, categories, center]);
 
     /**
      * Based on the combination of the states for venueName, locationId & kioskOriginLocationId,
      * determine where to make the map go to.
      */
     function determineMapBounds() {
         const currentVenue = venuesInSolution.find(venue => venue.name.toLowerCase() === currentVenueName.toLowerCase());
+
         if (mapsIndoorsInstance && currentVenue) {
             setMapPositionInvestigating(true);
 
+            // Parse center prop into coordinates. If it is not included in the URL, latLng are undefined.
+            const [latitude, longitude] = center
+                ? center.split(",").map(Number)
+                : [undefined, undefined];
+
+            // Create centerPoint object.
+            const centerPoint = { geometry: { type: 'Point', coordinates: [latitude, longitude] } };
+
+            // If startZoomLevel is not defined, fallback to 16 default value.
```

Comment:
This is written like it's a universal thing, but it's only related to when the center prop is set, right?